src/kirby/components/card/card.component.tns.html

```diff
@@ -1,4 +1,4 @@
-<FlexboxLayout (loaded)="onViewLoaded($event)" class="background" [ngClass]="cardSizeClass">
+<FlexboxLayout (loaded)="onViewLoaded($event)" class="background" [ngClass]="cardSizeClass" (tap)="select.emit()">
```

Comment:
We don't do it this way, make a function for that:
```suggestion
<FlexboxLayout (loaded)="onViewLoaded($event)" class="background" [ngClass]="cardSizeClass" (tap)="onCardSelect()">
```